Makefile

```diff
@@ -2,17 +2,36 @@ ARCH ?= $(shell uname -m)
 HOST_ARCH ?= $(shell uname -m)
 
 VERSION := v0.13.0
-TAG := bottlerocket/sdk-$(ARCH):$(VERSION)-$(HOST_ARCH)
-ARCHIVE := bottlerocket-sdk-$(ARCH)-$(VERSION).$(HOST_ARCH).tar.gz
 
-$(ARCHIVE) :
-	@DOCKER_BUILDKIT=1 docker build . -t $(TAG) --squash --build-arg ARCH=$(ARCH)
-	@docker image save $(TAG) | gzip --fast > $(@)
+SDK_TAG := bottlerocket/sdk-$(ARCH):$(VERSION)-$(HOST_ARCH)
+SDK_ARCHIVE := bottlerocket-sdk-$(ARCH)-$(VERSION).$(HOST_ARCH).tar.gz
 
-.PHONY: upload clean
+TOOLCHAIN_TAG := bottlerocket/toolchain-$(ARCH):$(VERSION)-$(HOST_ARCH)
+TOOLCHAIN_ARCHIVE := bottlerocket-toolchain-$(ARCH)-$(VERSION).$(HOST_ARCH).tar.xz
 
-upload : $(ARCHIVE)
-	@aws s3 cp $(ARCHIVE) s3://thar-upstream-lookaside-cache/$(TAG).tar.gz
+all: $(SDK_ARCHIVE) $(TOOLCHAIN_ARCHIVE)
+
+$(SDK_ARCHIVE) :
+	@DOCKER_BUILDKIT=1 docker build . \
+		--tag $(SDK_TAG) \
+		--target sdk-final \
+		--squash \
+		--build-arg ARCH=$(ARCH)
+	@docker image save $(SDK_TAG) | gzip --fast > $(@)
+
+$(TOOLCHAIN_ARCHIVE) :
+	@DOCKER_BUILDKIT=1 docker build . \
+		--tag $(TOOLCHAIN_TAG) \
+		--target toolchain-final \
+		--squash \
+		--build-arg ARCH=$(ARCH)
+	@docker run --rm --entrypoint cat $(TOOLCHAIN_TAG) /tmp/toolchain.tar.xz > $(@)
+
+upload : $(SDK_ARCHIVE) $(TOOLCHAIN_ARCHIVE)
```

Comment:
Seems like we might want a `download-toolchain` target now too?

Reply:
We'll handle the download and conversion into the "kmod kit" in Bottlerocket, since that's where we bring together the kernel and the toolchain.